backend/core/src/seed.ts

```diff
@@ -0,0 +1,59 @@
+import dotenv from "dotenv"
+import "reflect-metadata"
+import { createConnection } from "typeorm"
+import { Listing } from "./entity/Listing"
+import { Listing as OldListing } from "@bloom-housing/core"
+import listingsLoader from "./lib/listings_loader"
+import { Unit } from "./entity/Unit"
+import { Attachment } from "./entity/Attachment"
+import { Preference } from "./entity/Preference"
+
+const skipped = ["id", "units", "attachments", "preferences"]
+const types = { units: Unit, attachments: Attachment, preferences: Preference }
+let l
+createConnection()
+  // eslint-disable-next-line @typescript-eslint/require-await
+  .then(async connection => {
+    dotenv.config({ path: ".env" })
+    const listings = (await listingsLoader("listings")) as OldListing[]
+
+    for await (const listing of listings) {
+      l = new Listing()
+      const entities = {}
+      for await (const key of Object.keys(listing)) {
+        if (!skipped.includes(key)) {
+          l[key] = listing[key]
+        } else {
+          if (key != "id") {
```

Comment:
/pedantic `else if ()` & `!==` rather than `!=` (just JS best practices, doesn't change anything in this script)